test/passes/remove-unused-brs_all-features.wast

```diff
@@ -53,4 +54,58 @@
    (ref.func $i32_=>_none)
   )
  )
+
+ (func $br_on-to-br (param $func (ref func))
+  (call $log (i32.const 0))
+  (block $null
+   ;; a non-null reference is not null, and the br is never taken
+   (drop
+    (br_on_null $null (ref.func $br_on-to-br))
+   )
+   (call $log (i32.const 1))
```

Comment:
Is this `call $log` for debugging or they have a purpose here?

Reply:
Mainly just to make it easy to debug and read the testcase, to easily see what matches what.